src/Maker/MakeAuthenticator.php

```diff
@@ -55,88 +71,275 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
+    {
+        $controllerClassNameDetails = $this->generator->createClassNameDetails(
+            $controllerClass,
+            'Controller\\',
+            'Controller'
+        );
+
+        if (!class_exists($controllerClassNameDetails->getFullName())) {
+            $controllerPath = $this->generator->generateController(
+                $controllerClassNameDetails->getFullName(),
+                'authenticator/EmptySecurityController.tpl.php'
+            );
+
+            $controllerSourceCode = $this->generator->getFileContentsForPendingOperation($controllerPath);
+        } else {
+            $controllerPath = $this->fileManager->getRelativePathForFutureClass($controllerClassNameDetails->getFullName());
+            $controllerSourceCode = $this->fileManager->getFileContents($controllerPath);
+        }
+
+        if (method_exists($controllerClassNameDetails->getFullName(), 'login')) {
+            throw new RuntimeCommandException(sprintf('Method "login" already exists on class %s', $controllerClassNameDetails->getFullName()));
+        }
+
+        $manipulator = new ClassSourceManipulator($controllerSourceCode, true);
+
+        $securityControllerBuilder = new SecurityControllerBuilder();
+        $securityControllerBuilder->addLoginMethod($manipulator);
+
+        $this->generator->dumpFile($controllerPath, $manipulator->getSourceCode());
+
+        // create login form template
+        $this->generator->generateFile(
+            'templates/security/login.html.twig',
+            'authenticator/login_form.tpl.php',
+            [
+                'username_field' => $userNameField,
+            ]
+        );
+    }
+
+    private function generateNextMessage(bool $securityYamlUpdated, string $authenticatorType, string $authenticatorClass, array $securityData, $userClass): array
+    {
+        $nextTexts = ['Next:'];
+        $nextTexts[] = '- Customize your new authenticator.';
+
         if (!$securityYamlUpdated) {
             $yamlExample = $this->configUpdater->updateForAuthenticator(
                 'security: {}',
                 'main',
                 null,
-                $classNameDetails->getFullName()
+                $authenticatorClass
             );
-            $text[] = "Your <info>security.yaml</info> could not be updated automatically. You'll need to add the following config manually:\n\n".$yamlExample;
+            $nextTexts[] = '- Your <info>security.yaml</info> could not be updated automatically. You\'ll need to add the following config manually:\n\n'.$yamlExample;
+        }
+
+        if (self::AUTH_TYPE_FORM_LOGIN === $authenticatorType) {
+            $nextTexts[] = sprintf('- You must provide a valid redirection in the method <info>%s::onAuthenticationSuccess()</info>.', $authenticatorClass);
+            $nextTexts[] = '- Review & adapt the login template : <info>/templates/security/login.html.twig</info>.';
```

Comment:
> - Review & adapt the login template: <info>templates/security/login.html.twig</info>.

Reply:
And let's move this down below the next two. That way, if there are multiple things you need to do in the authenticator, we should all of them, and THEN the template text